Description: Show gnomecc.desktop, so that we can find it in Unity. For classic GNOME we will disable it in /etc/xdg/menus/settings.menu instead.
Author: Martin Pitt <martin.pitt@ubuntu.com>
Bug-Ubuntu: https://launchpad.net/bugs/727823

Index: gnome-control-center-2.32.1/shell/gnomecc.desktop.in.in
===================================================================
--- gnome-control-center-2.32.1.orig/shell/gnomecc.desktop.in.in	2011-03-07 15:33:18.807496658 +0100
+++ gnome-control-center-2.32.1/shell/gnomecc.desktop.in.in	2011-03-07 15:33:21.687496659 +0100
@@ -12,4 +12,3 @@
 X-GNOME-Bugzilla-Product=gnome-control-center
 X-GNOME-Bugzilla-Component=shell
 X-GNOME-Bugzilla-Version=@VERSION@
-NoDisplay=true
